app/src/androidTest/java/com/github/sdp/mediato/DatabaseTests/CollectionsTests.java

```diff
@@ -41,18 +30,21 @@
 public class CollectionsTests {
     private final static int STANDARD_COLLECTION_TIMEOUT = 10;
     private User user1 = new User.UserBuilder("uniqueId1")
-            .setUsername("user_test_1")
-            .setEmail("email_test_1")
-            .setRegisterDate("09/03/2023")
-            .setLocation(new Location(3.14, 3.14))
-            .build();
+        .setUsername("user_test_1")
+        .setEmail("email_test_1")
+        .setRegisterDate("09/03/2023")
+        .setLocation(new Location(3.14, 3.14))
+        .build();
 
     private Collection collection1;
     private Collection collection2;
 
-    private Review review1 = new Review(user1.getUsername(), new Media(MediaType.MOVIE, "Harry Potter 1", "the chosen one", "url", 1));
-    private Review review2 = new Review(user1.getUsername(), new Media(MediaType.MOVIE, "Harry Potter 2", "the chosen two", "url", 2), 9);
-    private Review review3 = new Review(user1.getUsername(), new Media(MediaType.MOVIE, "Harry Potter 3", "the chosen three", "url", 3), 2, "meh");
+    private Review review1 = new Review(user1.getUsername(),
+        new Movie("Harry Potter 1", "the chosen one", "url", 1));
+    private Review review2 = new Review(user1.getUsername(),
+        new Movie("Harry Potter 2", "the chosen two", "url", 2), 9);
+    private Review review3 = new Review(user1.getUsername(),
+        new Movie("Harry Potter 3", "the chosen three", "url", 3), 2, "meh");

```

Comment:
Since almost only the constructor with 2 image strings will be used (if I am not mistaken), it might be good to use it in our tests.

Reply:
Which constructor do you mean? `Movie` only has one other constructor that takes a `TMDBMovie`

Reply:
Ok I thought that some changes were already merged on main but there are still not merged. My bad.